Client/Frontend/Strings.swift

```diff
@@ -56,7 +56,28 @@ private func MZLocalizedString(
 // MARK: - Alerts
 extension String {
     public struct Alerts {
-
+        public struct RestoreTabs {
+            public static let Title = MZLocalizedString(
+                "Alerts.RestoreTabs.Title.v109",
+                tableName: "Alerts",
+                value: "Firefox crashed. Restore your tabs?",
+                comment: "The title of the restore tabs pop-up alert.")
+            public static let Message = MZLocalizedString(
+                "Alerts.RestoreTabs.Message.v109",
+                tableName: "Alerts",
+                value: "Sorry about that. Restore tabs to pick up where you left off.",
+                comment: "The body of the restore tabs pop-up alert.")
+            public static let ButtonNo = MZLocalizedString(
+                "Alerts.RestoreTabs.Button.No.v109",
+                tableName: "Alerts",
+                value: "No",
+                comment: "The title for the negative action of the restore tabs pop-up alert")
+            public static let ButtonYes = MZLocalizedString(
+                "Alerts.RestoreTabs.Button.Yes.v109",
+                tableName: "Alerts",
+                value: "Restore tabs",
+                comment: "The title for the affirmative action of the restore tabs pop-up alert")
```

Comment:
Also seems pretty straight forward, given the string and the comment together make things pretty explicit.
But we can still add a little bit just in case, like "The title for the affirmative action of the restore tabs pop-up alert. This alert appears when Firefox crashes"